app/forms/mappings/Formatters.scala

```diff
@@ -128,46 +129,46 @@ trait Formatters {
     }
 
   private[mappings] def currencyFormatter(
-    requiredKey: String,
-    invalidNumericKey: String,
-    nonNumericKey: String,
-    decimalCount: Int,
-    args: Seq[String] = Seq.empty
-  ): Formatter[BigDecimal] =
+                                           requiredKey: String,
+                                           invalidNumericKey: String,
+                                           nonNumericKey: String,
+                                           decimalCount: Int,
+                                           args: Seq[String] = Seq.empty
+                                         ): Formatter[BigDecimal] =
     new Formatter[BigDecimal] {
```

Comment:
do we use this anywhere 🤔

Reply:
It was used for ConvertedCredit and might be when we bring it back

Reply:
sure we'll leave it for now, but i think credit will end up being a long